pytorch/training/docker/2.1/py3/Dockerfile.cpu

```diff
@@ -114,10 +114,11 @@ RUN /opt/conda/bin/mamba install -y --override-channels \
     -c conda-forge \
     conda=23.1.0 \
     # patch CVE
-    "cryptography>=41.0.4" \
+    "cryptography>=42.0.2" \
+    # patch CVE
+    "pyopenssl>=21.0.0" \
```

Comment:
For this as well.

Reply:
For CPU it was `24.0.0` from the [logs here](https://us-west-2.console.aws.amazon.com/codesuite/codebuild/projects/dlc-pr-sanity-test/build/dlc-pr-sanity-test%3A4c0e8483-c49d-4621-8f1d-9677a4617d44/log?region=us-west-2).